ci/tasks/http-ping.yml

```diff
@@ -0,0 +1,19 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/curl-ssl
+    tag: fe3e384e81ccb50842509d7237e3828b293de694
+params:
+  URL:
+run:
+  path: sh
+  args:
+    - -euxc
+    - |
+      DOMAIN=$(echo "${URL}" | awk -F/ '{print $3}')
+      getent ahosts ${DOMAIN} | cut -d ' ' -f1 | sort | uniq | xargs -n1 -I {} curl \
```

Comment:
I have no idea what this line does. I'm sure you've tested it and it works a treat! :)

Reply:
yeah name service voodoo ... it's like a poor man's `dig`